### ISSUE TITLE
Changing the name in Home App results in lost connection

### ISSUE THREAD

**Issue by chrispi68** [grzegorz914/homebridge-denon-tv#35]
Hi,

When I change the namen of the Denon device in the iOS Home App and restart HomeBridge, the device is marked with "No Response". To solve this, I have to delete the device a add it again. To add it, I have to give it a new name in the config because otherwise the devide does not show up.

Is this a known problem? Other device in my homebride instance (more than 40) do not behave like this.

When I keep the name in the Confg file and in the Home app the same, there is no problem.

Best regards
Christoph

**Comment by grzegorz914** [comment]
Are the other devices register as Dynamic External Platform Accessory?

**Comment by chrispi68** [comment]
Yes, partly. 25 are single accessories and the rest ist provided via platform.

**Comment by grzegorz914** [comment]
Provide via bridge platform or external dynamic platform?

**Comment by chrispi68** [comment]
It’s a mix.
